apps/frontend/src/components/global/groups/GroupUsers.vue

```diff
@@ -0,0 +1,77 @@
+<template>
```

Comment:
This seems very similar to `src/components/global/groups/Users.vue`. Could the same component be reused with some deduplication?